src/carp.cc

```diff
@@ -58,7 +58,9 @@ carpInit(void)
     /* find out which peers we have */
 
     RawCachePeers rawCarpPeers;
-    for (auto p = Config.peers; p; p = p->next) {
+    for (const auto &peer: CurrentCachePeers()) {
+        const auto p = peer.get();
+
         if (!cbdataReferenceValid(p))
             continue;
```

Comment:
These two lines from the official code will be gone after #1454 is merged. We can remove them now, proactively, but that would be an out-of-scope change and it may create a merge conflict that would otherwise may be avoided.

Reply:
... said merge has happened.